README.md

```diff
@@ -1,3 +1,18 @@
+# libcangjie
+
+[![travis-shield]][travis-url]
+[![docker-status-shield]][docker-url]
+[![docker-automate-shield]][docker-url]
```

Comment:
I'm not sure we really should have the 2 Docker-related badges: the Docker images are really just an implementation detail of our CI, whereas these badges feels like the Docker images are a primary deliverable we build and publish as part of working on libcangjie.

Reply:
Let's remove the automate shield then.

Reply:
Since we have docker image, I do think it is nice to have at least 1 shield here. Who knows. People may want to build a Python based Cangjie web service on our docker image :-)

At least it is good for us monitoring the build status in Dockerhub.

Reply:
> People may want to build a Python based Cangjie web service on our docker image :-)

And they really shouldn't.

It's an image built for CI, and as such is entirely unsuitable for production use:

* it builds libcangjie master
* it's way too big, including a compiler, a build toolchain, etc...

Reply:
> At least it is good for us monitoring the build status in Dockerhub.

Indeed.

Do we receive emails when the automatic builds fail?

Reply:
You can enable that at:
https://hub.docker.com/account/notifications/

Reply:
OK. Let's remove all Docker shields then.

Reply:
Ah, very cool, I had missed the notifications setting, thanks!

> OK. Let's remove all Docker shields then.

I think it would be very useful if we provided production-ready images.

However I am entirely uninterested in providing such images, and won't do any work towards that. I won't prevent anybody from doing it, though. 🙂